vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -174,6 +228,9 @@ def parse_base_item(item)
       {
         :ems_ref            => item.metadata.uid,
         :name               => item.metadata.name,
+        # namespace is overriden in more_core_extensions and hence needs
+        # a non method access
```

Comment:
@brandondunne Can we remove namespace at the instance level?  I too find this a terrible method there.  Class method should be fine, but the instance method gets in the way.